lib/sensors/temperature.hpp

```diff
@@ -11,16 +11,14 @@
 
 namespace hyped::sensors {
 
-// Two possible addresses for the temperature sensor
-constexpr std::uint8_t kDefaultTemperatureAddress     = 0x38;
-constexpr std::uint8_t kAlternativeTemperatureAddress = 0x3F;
+enum class temperatureAddress { k7F = 0x7F, k38 = 0x38, k3C = 0x3C, k3E = 0x3E };
```

Comment:
According to the datasheet, the addresses are 0x70/78/7C/7E

Reply:
uh where are you seeing that? the sensor definitely shows up at 0x38